deploy-deps.sh

```diff
@@ -8,6 +8,7 @@ main() {
     echo "Testing PVC creation for default storage class" >&2
     test_pvc_binding
     echo "Deploying Konflux Dependencies" >&2
+    install_argocd
```

Comment:
Should it be a part of the `deploy-deps.sh` script?

Reply:
The script name is misleading. 
It does not install ArgoCD.

Reply:
I agree. deploy-deps.sh shouldn't have any changes introduced by this PR.